src/solc_0.8/avatar/AvatarBase.sol

```diff
@@ -0,0 +1,68 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {ContextUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
```

Comment:
Not sure all imports are needed as ERC721Upgradeable is already Initializable and ContextUpgradeable

Reply:
It doesn't make any harm and this make it explicit.